src/components/ADempiere/ContextMenu/contextMenuDesktop.vue

```diff
@@ -35,12 +35,32 @@
               {{ $t('components.contextMenuDownload') }}
             </a>
           </el-menu-item>
-          <el-menu-item v-if="getDataSelection.length > 0 && panelType === 'browser'" index="6" @click="exporBrowser">
-            {{ $t('components.contextMennuExport') }}
-          </el-menu-item>
-          <el-menu-item v-if="panelType === 'window'" index="7" @click="exporBrowser">
-            {{ $t('components.contextMennuWindowReport') }}
-          </el-menu-item>
+          <el-submenu
+            v-if="getDataSelection.length > 0 && panelType === 'browser'"
+            index="xlsx"
+            @click.native="exporBrowser('xlsx')"
+          >
+            <template slot="title">{{ $t('components.contextMennuWindowReport') }}</template>
+            <el-menu-item index="xlsx">xlsx</el-menu-item>
```

Comment:
@elsiosanchez please change it for a util class using a supported export types and remove hardcoded values

A example:
```Javascript
let supportedTypes = {
  'xls',
  'txt'
}
```